config/initializers/assets.rb

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-Rails.application.config.assets.precompile << 'solidus_frontend_manifest.js'
+Rails.application.config.assets.precompile << 'solidus_legacy_frontend_manifest.js'
```

Comment:
Isn't it going to break applications using `solidus_frontend` now? If so, we could just keep the old name.

Reply:
I'm not sure if it will break apps in productions. I'll test it first.

Reply:
I retained the SolidusFrontend name in the new PR update.